src/base.py

```diff
@@ -28,5 +28,16 @@ def change_to_base(n: int, b: int) -> str:
     >>> change_to_base(31, 16)
     '1F'
     """
+    
+    final_list = []
     assert 2 <= b <= 16
-    return ''  # FIXME: return n in the right base
+    if n == 0:
+        return "0"
+    if n > 0:
+        final_list.append(n % b)
+        n = n//b
+    if n < 0:
+        negative_list.append(n % b)
```

Comment:
If you run this, you will notice that `negative_list` doesn't exist, so you get an error. You did run the code before you submitted it and asked for a review, right? ;) 

You have the same issue as for positive numbers, though. You only handle one digit, and you need them to be strings at some point.